css/src/components/input.css

```diff
@@ -19,15 +19,19 @@
 }
 
 .Input--tiny {
-  height: var(--spacing-xl);
+  height: var(--font-height-m);
   padding-top: var(--spacing-m);
   padding-bottom: var(--spacing-m);
 }
 
 .Input--regular {
-  height: var(--spacing-3);
-  padding-top: var(--spacing);
-  padding-bottom: var(--spacing);
+  height: var(--font-height-l);
+  padding-top: 6px;
+  padding-bottom: 6px;
+}
+
+.Input--regular .Input-input {
+  line-height: var(--font-height);
```

Comment:
Please try to apply the css directly to `Input-input--regular`

Reply:
Done